tools/configen/configen/configen.py

```diff
@@ -128,15 +128,45 @@ def is_incompatible(type_: Type[Any]) -> bool:
     return True
 
 
+def get_default_flags(module: ModuleConf) -> List[Parameter]:
+    flag_type_dict = {
+        "_convert_": "str",
+        "_recursive_": "bool",
+    }
+
+    def_flags: List[Parameter] = []
+    for name, default in module.default_flags.items():
+        try:
+            flag_type = flag_type_dict[name]
+        except KeyError:
+            log.error(f"Flag '{name}' unsupported by configen.")
+            raise
+        default = f'"{default}"' if flag_type == "str" else default
+
+        def_flags.append(
+            Parameter(
+                name=name,
+                type_str=flag_type,
+                default=default,
+            )
+        )
+
+    return def_flags
+
+
 def generate_module(cfg: ConfigenConf, module: ModuleConf) -> str:
     classes_map: Dict[str, ClassInfo] = {}
     imports = set()
     string_imports: Set[str] = set()
+
+    default_flags = get_default_flags(module)
+
```

Comment:
So much cleaner thank trying to force it into an already too large function, isn't it? :)